src/util/wallet-utils/src/sign-psbt.ts

```diff
@@ -36,6 +39,21 @@ export async function signPSBTXverse({ hex, address }: SignPSBTParams) {
   return base64ToHex(response.result.psbt);
 }
 
+export async function signPSBTFordefi({ hex, address }: SignPSBTParams) {
+  const provider = await getFordefiBTCProviderOrThrow();
+  const response = await provider.unisatProvider.signOrSignAndSendPsbt(hex, {
```

Comment:
sanity check: is that `unisatProvider` correct here? Also I've got no clue where to look for `signOrSignAndSendPsbt`

Reply:
neither do I, no docs for this behavior unfortunately, I'm in the dark here but this seems to be working